### ISSUE TITLE
Please give credit to the real author(s) (including me).

### ISSUE THREAD

**Issue by rxzyx** [DavidKitKat/Blooket-Hacks#4]
I noticed that the hacks aren't made by you, can you please give credit to the author(s) including me? Thanks.

**Comment by qaiik** [comment]
Same. I made some of these hacks. I saw they where merged into. Hp7’s repo and I got credit there though.

**Comment by glixxzzy** [comment]
Yeah.. Can you give credit to all of us? You made pull requests and forks, give us credit or I'm going to submit a DMCA takedown report.

**Comment by DavidKitKat** [comment]
ok i did
